android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -185,6 +185,11 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
         }
     }
 
+    @ReactMethod
+    override fun validatePinWithPolicy(pin: String, promise: Promise) {
```

Comment:
shouldn't pin parameter be nullable? Won't it crash when developer passes null pin from RN? (I don't know if RN does any null check when passing values to native)

Reply:
I believe it will crash and we will need to change this for all other methods aswell.

Some android Native methods however don't accept nullable parameters, e.g. `getAllAuthenticators(@NonNull UserProfile userProfile)` I think we can repurpose the added PARAMETERS_NOT_CORRECT error for methods which require a NonNull parameter and then remove the specific USER_PROFILE_IS_NULL error. 

But in this case, it is indeed correct to just let the SDK handle it.